### PR TITLE
Added paypal's new password shitty requirements

### PR DESCRIPTION
**Pull request by zekchopper**
Previous PayPal entry doesn't shame them enough.

### CODE REVIEW

**Comment by duffn** [comment]
Where does that screenshot come from? Is that on mobile? Is that the new password message?

**Comment by zekchopper** [comment]
No, just the regular site.
https://www.paypal.com/welcome/signup
The message keeps you updated on which particular asinine rules you are currently breaking as you're writing your password.

**Comment by duffn** [comment]
Cool, I see it now. If this is the new message, can you actually replace the old image with this one?

**Comment by zekchopper** [comment]
Done.

File: README.rst [hunk 1]
```diff
@@ -1828,6 +1827,7 @@ The new Password cannot be the same as the last 32 passwords you have used. (the
 .. |PagoMisCuentas| image:: /screenshots/pagomiscuentas.png
 .. |Parnassus| image:: /screenshots/parnassus.png
 .. |PayPal| image:: /screenshots/paypal.png
+.. |PayPalNewPassword| image:: /screenshots/paypalnewpassword.png
```
Comment (duffn) [inline README.rst]: Remove this too, please.

Comment (zekchopper) [reply README.rst]: sure missed that

**Comment by duffn** [comment]
Thank you!

**Pull request merged**